core/plugin/zplugin.go

```diff
@@ -56,6 +56,7 @@ import (
 	_ "github.com/coredns/coredns/plugin/tsig"
 	_ "github.com/coredns/coredns/plugin/view"
 	_ "github.com/coredns/coredns/plugin/whoami"
+
```

Comment:
Hack to pass the `TestImportOrdering` test. Otherwise unit test fails.
External plugin will mostly not belong to `github.com/coredns` which causes the `TestImportOrdering` to fail.